src/components/EducationCard.tsx

```diff
@@ -38,7 +38,7 @@ export function WalletGraphic({ quantity, verb = "retained" }: EducationGraphicP
       </Box>
       <Box display="flex" flex="1" alignItems="center" alignContent="center" justifyContent="center" className="text">
         <Typography variant="h6" align="center" className="cta-text">
-          {Number(quantity).toFixed(4)} sOHM {verb}
+          {quantity} sOHM {verb}
```

Comment:
can you trim this though? my wallet is 68.282983123 sOHM

Reply:
Checking

Reply:
It was supposed to be 4 decimal points